server/app/boot_jobs.rb

```diff
@@ -2,9 +2,12 @@
 require_relative 'services/worker_supervisor'
 require_relative 'services/mongodb/migrator'
 
-unless ENV['RACK_ENV'] == 'test' || ENV['NO_MONGO_PUBSUB']
-  MongoPubsub.start!(PubsubChannel)
-
+unless ENV['RACK_ENV'] == 'test'
+  if servers = ENV['NATS_SERVERS']
+    MasterPubsub.start!(servers.split(','))
```

Comment:
The type confusion going on between the two `MasterPubsub.start!` method calls here isn't very nice. I can see what's going on now (the `MasterPubsub` class is defined programatically), but...

Reply:
I agree, will fix.